bintray.gradle

```diff
@@ -52,11 +38,6 @@ bintray {
     version{
       name = project.version
       vcsTag = 'v' + project.version
-      mavenCentralSync{
```

Comment:
Added commands for automatic publishing, so this doesn't seem necessary anymore.